.github/workflows/checksums.yml

```
@@ -0,0 +1,64 @@
+name: Compute checksums
+
+on:
+  push:
+    branches: [ checksums ]
+
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: 'The tag name for which to run it (eg: v22.1.0). If not specified, defaults to the latest release'
+        required: false
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+
+    - name: Download binaries
+      shell: bash
+      run: |
+        set -x
+        TAG_NAME=${{ github.event.inputs.tag }}
+        if [ -z "$TAG_NAME" ]; then
+          url=https://api.github.com/repos/NREL/EnergyPlus/releases/latest
+        else
+          url=https://api.github.com/repos/NREL/EnergyPlus/releases/tags/$TAG_NAME
+        fi;
+
+        echo "Querying url=$url"
+        release_info=$(curl -sL $url)
+
+        # Get the tag name, and use jq -e to throw if can't find
+        TAG_NAME=$(echo $release_info | jq -r -e '.tag_name')
+        echo "TAG_NAME=$TAG_NAME" >> $GITHUB_ENV
+
+        echo $release_info | jq -r '.assets[].browser_download_url' > download_lists.txt
```

I suppose in case we run it **twice** in the row on the same tag / latest, we want to filter out the already existing md5sum.txt and sha256sums.txt
```suggestion
        echo $release_info | jq -r '.assets[].browser_download_url | select(. | contains("EnergyPlus-"))' > download_lists.txt
```

---

Well https://github.com/svenstaro/upload-release-action already won't override, but better be safe

> overwrite: If an asset with the same name already exists, overwrite it (Default: false).